app/api/pet_chat.py

```
@@ -1,12 +1,65 @@
+        return {"pets": pets}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching pets: {str(e)}")
+
+@llm_api.post("/chat/{pet_id}")
+async def chat_with_pet(pet_id: int, message: dict):
+    """MCP로 펫 정보를 조회하고 채팅 응답 생성"""
+    try:
+        # MCP를 통해 펫 정보 조회
+        pet_info = await get_pet_info(pet_id)
+        if not pet_info:
+            raise HTTPException(status_code=404, detail="Pet not found")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)